.github/scripts/package_and_deploy.sh

```diff
@@ -6,8 +6,9 @@ ddprof_deploy_folder=$1
 commit_sha=$2
 commit_author=$3
 
-
-profiler_version=v2.5.0.0_$(date -u +%G%m%d%H%M%S)
+current_profiler_version=$(sed -rn 's/constexpr auto PROFILER_VERSION = "([^"]+)";/\1/p' ../profiler/src/ProfilerEngine/Datadog.Profiler.Native/dd_profiler_version.h)
```

Comment:
Interesting, this one wasn't well bumped. Have you checked if we have forgotten other instances in the VersionBump process?

But also, shouldn't we just add this version in the [bump process](https://cs.github.com/DataDog/dd-trace-dotnet/blob/a908775dbf6c402aa45f19c447aa252cce0b7754/tracer/build/_build/PrepareRelease/SetAllVersions.cs?q=SetAllVersions#L41) instead of doing a sed?

Reply:
Yep that one the last (🤞 ) occurrence.

>But also, shouldn't we just add this version in the [bump process](https://cs.github.com/DataDog/dd-trace-dotnet/blob/a908775dbf6c402aa45f19c447aa252cce0b7754/tracer/build/_build/PrepareRelease/SetAllVersions.cs?q=SetAllVersions#L41) instead of doing a sed?

We can, but I will still need to do a `sed` to get my `PROFILER_BETA_REVISION`. So I prefer having one place for the bump, and the rest just extract it from there.